test/recrypt-node.test.js

```diff
@@ -466,4 +466,30 @@ describe("Recrypt-Node", () => {
             expect(transformKeyBytes).toHaveLength(672);
         });
     });
+
+    describe("addPrivateKeys", () => {
+        it("should add together the provided keys", () => {
+            //prettier-ignore
+            const key1 = Buffer.from([1, 2, 0, 221, 116, 9, 241, 149, 253, 82, 219, 45, 60, 186, 93, 114, 202, 103, 9, 191, 29, 148, 18, 27, 243, 116, 136, 1, 180, 1, 1, 0]);
```

Comment:
Did you pull these keys from some other place? Just wondering how you verified them.

Reply:
I just copied them from the tests in IronWeb. I don't exactly recall how I did them there, but I likely just generated both of them via Recrypt.